decidim-admin/spec/commands/update_organization_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+require "spec_helper"
+
+module Decidim
+  module Admin
+    describe UpdateOrganization, :db do
+      describe "call" do
+        let(:organization) { create(:organization) }
+        let(:form) do
+          OrganizationForm.from_params(
+            organization: organization.attributes.merge(name: "My super organization"),
```

Comment:
I mean, instead of `organization.attributes.merge` just write down everything we need to pass to the form.